From 5215d437cca55e2b5ed58ae954e4a27d7afcebab Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Mon, 15 Aug 2016 23:19:00 +0200
Subject: [PATCH] Include glib.h instead of individual headers

---
 sms-glib/errors.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sms-glib/errors.c b/sms-glib/errors.c
index 73ca3d9..e7835c8 100644
--- a/sms-glib/errors.c
+++ b/sms-glib/errors.c
@@ -22,7 +22,7 @@
 #include "config.h"
 
 #include <sms-glib/errors.h>
-#include <glib/gstrfuncs.h>
+#include <glib.h>
 
 #include <stdarg.h>
 
