Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -99,7 +99,7 @@ fi
 
 AC_CHECK_PROGS([XSLTPROC], [$XSLTPROC xsltproc])
 AC_CHECK_PROGS([DBUS_BINDING_TOOL], [$DBUS_BINDING_TOOL dbus-binding-tool])
-AC_CHECK_PROGS([PYTHON], [$PYTHON python2.5 python2.4 python2.3 python])
+AC_CHECK_PROGS([PYTHON], [$PYTHON python2 python2.5 python2.4 python2.3 python])
 
 AM_CONDITIONAL([HAVE_TP_EXTENSIONS], [test -n "$XSLTPROC" && test -n "$PYTHON" && test -n "$DBUS_BINDING_TOOL"])
 
